addons/l10n_in_edi_ewaybill/views/account_move_views.xml

```diff
@@ -16,7 +16,7 @@
                     <group name="ewaybill_group">
                         <group string="Transaction Details" name="Transaction_group" 
                             invisible="not l10n_in_edi_ewaybill_direct_api">
-                            <field name="l10n_in_type_id" domain="[('allowed_supply_type', 'in', ('in', 'both'))] if move_type in ('in_invoice', 'in_refund', 'in_receipt') else [('allowed_supply_type', 'in', ('out', 'both'))]"/>
+                            <field name="l10n_in_type_id" domain="[('code', '!=', 'CHL'), ('allowed_supply_type', 'in', ('in', 'both'))] if move_type in ('in_invoice', 'in_refund', 'in_receipt') else [('code', '!=', 'CHL'), ('allowed_supply_type', 'in', ('out', 'both'))]"/>
```

Comment:
We can not have CHL, because for CHL you need a linked invoice?

Reply:
Yupp
Challan Type - would only be for stock